tslint.json

```diff
@@ -5,6 +5,7 @@
     "tslint-react"
   ],
   "rules": {
+    "ordered-imports": [false],
```

Comment:
Why?

Reply:
I would argue that imports shouldn't be ordered by alphabetasion but grouped on "import type"
ex: dependency-imports should be grouped together, imported components within your projects in one grouping, imported functions/constants in a grouping.